account_asset_number/models/account_asset_profile.py

```diff
@@ -17,7 +17,28 @@ class AccountAssetProfile(models.Model):
         string="Asset Number Sequence",
         domain=lambda self: self._get_domain_sequence_id(),
     )
+    barcode_type = fields.Selection(
+        selection=[("barcode", "Barcode"), ("qr", "QR")],
+        default="barcode",
+    )
+    barcode_width = fields.Integer(
+        default=350,
+        help="Width of barcode or QR code",
```

Comment:
And the same for the other

```suggestion
        help="Width (in px) of the barcode or the QR code",
```